src/backend/base/langflow/services/database/service.py

```diff
@@ -53,6 +55,23 @@ def _create_engine(self) -> "Engine":
             max_overflow=self.settings_service.settings.max_overflow,
         )
 
+    @event.listens_for(Engine, "connect")
+    def on_connection(dbapi_connection, connection_record):
+        from sqlite3 import Connection as sqliteConnection
+
+        if isinstance(dbapi_connection, sqliteConnection):
+            logger.info("sqlite connect listener, setting pragmas")
+            cursor = dbapi_connection.cursor()
+            try:
+                cursor.execute("PRAGMA synchronous = NORMAL")
+                cursor.execute("PRAGMA journal_mode = MEMORY")
+                cursor.execute("PRAGMA cache_size = 10000")
+                cursor.execute("PRAGMA temp_store = MEMORY")
+                cursor.execute("PRAGMA locking_mode = EXCLUSIVE")
```

Comment:
Maybe we should remove this because of FastAPI being async and all that. What do you think?

Reply:
Thanks @ogabrielluiz I reduced only these two config for faster write to sqlite on EFS
```
                cursor.execute("PRAGMA synchronous = NORMAL")
                cursor.execute("PRAGMA journal_mode = WAL")
```